metricbeat/module/postgresql/_meta/kibana/7/dashboard/Metricbeat-Postgresql-overview.json

```
@@ -0,0 +1,737 @@
+{
+  "objects": [
+    {
+      "attributes": {
+        "description": "",
+        "kibanaSavedObjectMeta": {
+          "searchSourceJSON": {
+            "filter": [],
+            "index": "metricbeat-*",
+            "query": {
+              "language": "lucene",
+              "query": ""
+            }
+          }
+        },
+        "title": "[Postgresql] Number of deadlock over time",
```

Visualization titles of this dashboard should have `[Metricbeat Postgresql]` as suffix
```suggestion
        "title": "Number of deadlock over time [Metricbeat Postgresql]",
```